critiquebrainz/frontend/external/musicbrainz_db/testing.py

```diff
@@ -0,0 +1,12 @@
+import os
+from flask_testing import TestCase
+from critiquebrainz.frontend import create_app
+
+class MBDatabaseTestCase(TestCase):
```

Comment:
Yes, `flask_testing`'s TestCase is required as it sets up the `critiquebrainz_redis` container for the tests. I get the following error when using `unittest.TestCase` > https://gist.github.com/ferbncode/2a3f3967015747c45efedfc20b7f3608